tensorflow/compiler/mlir/xla/transforms/hlo_legalize_to_lhlo.cc

```diff
@@ -205,8 +205,14 @@ struct HloToLhloReduceOpConverter : public BaseOpConversion<xla_hlo::ReduceOp> {
         entry_block.getArgument(original_arg_count).getType());
     // Remove the old arguments.
     for (int i = original_arg_count - 1; i >= 0; --i) {
-      entry_block.eraseArgument(i);
+//      entry_block.eraseArgument(i);
+      entry_block.eraseArgument(2*original_arg_count -1);
```

Comment:
This should be outside of the loop since it does not depend on the loop counter?